test/sql/window/test_order_by_all.test

```diff
@@ -14,3 +14,12 @@ FROM (
 ) t;
 ----
 Cannot ORDER BY ALL in a window expression
+
+# We CAN order by column patterns
+query II
+SELECT
+    rank() OVER (ORDER BY COLUMNS('^(.*)_score$') DESC) AS '\1_rank'
```

Comment:
Can we add a test that verifies this is actually correctly ordering? IIRC we disabled `ORDER BY ALL` because it was quietly ignored